resources/aws/securitygroup.go

```diff
@@ -0,0 +1,77 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+var (
+	portsToOpen []int = []int{
+		// SSH
+		22,
+		// Insecure apiserver
+		80,
+		// apiserver
+		443,
```

Comment:
We were using 6443 in #70, and that's what I see [elsewhere too](https://github.com/giantswarm/kubernetesd/blob/491a7c6cc47706dacf3b54d9027fa297b7bb4c94/kubernetes/configmap.yaml#L45). Which is it?